components/IngredientSelector/Navigator.tsx

```diff
@@ -130,9 +135,44 @@ export default function Navigator() {
           );
         })}
       </InteractionWrapper>
+    
     );
   }
-  return (
+
+     let value = "";
+      if (currentIngredient) {
+        const ingredient = data.ingredients[currentIngredient.ingredient_id];
+  
+        if (ingredient.category_id === "fat") {
+          if (ingredient.ingredient_name === "Huile d'olive") {
```

Comment:
As @alexfauquette said, we cannot hardcode each ingredient id and name to specify the quantity, instead we need to use the data in ingredients_taxonomy.json , and use the new "Quantity displayed" column in the file https://docs.google.com/spreadsheets/d/1aqYX47vkJnjiKVqLku_-f8IlbiFkjwpkgFyGGTSDfdI/edit#gid=0